src/Content/Backend/Solution/Monaco.Template.Backend.IntegrationTests/AccessTokenDto.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Monaco.Template.Backend.IntegrationTests;
+
+public record AccessTokenDto
+{
+	public AccessTokenDto(string accessToken, int expiresIn)
```

Comment:
Currently `expiresIn` is not being set or used?

Reply:
Yeah, I was fixing that, which is from something previous that I removed prior to the commit and it's breaking the execution.

Reply:
Fixed. Redundant parameter removed.